phpBB/styles/prosilver/theme/links.css

```diff
@@ -90,13 +90,15 @@ a.lastsubject:hover {
 }
 
 .row-item a:hover {
-	text-decoration: none
+	text-decoration: none;
 }
 
+.row-item .forumtitle:hover,
```

Comment:
@hanakin this is the part that i'm mainly fixing line 96 to 101

Reply:
Yes but this still goes against our guidelines. We try to keep prs focused on fixing a single thing to allow for easier merging and undoing. This should be split up into separate prs.

Reply:
Do a pr for just that fix, then a new one for the ;

Reply:
ok will do i'll just close this one cause that's easier for me and start the new ones